prog/dftb+/lib_dftbplus/parser.F90

```diff
@@ -3703,6 +3711,55 @@ subroutine readSolvGB(node, geo, input)
   end subroutine readSolvGB
 
 
+  subroutine readCM5(node, input, geo)
+
+    !> Node to process
+    type(fnode), pointer :: node
+
+    !> Geometry of the current system
+    type(TGeometry), intent(in) :: geo
+
+    !> Contains the input for the dispersion module on exit
+    type(TCM5Input), intent(out) :: input
+
+    type(fnode), pointer :: value1, child, child2, field
+    type(string) :: buffer, modifier
+    real(dp), allocatable :: atomicRadDefault(:)
+    integer :: iSp
+
+    call getChildValue(node, "Alpha", input%alpha, 2.474_dp/AA__Bohr, child=child)
```

Comment:
Bohr__AA is defined as a constant, so would save a division

Reply:
Since it is one over Ångström I find the division clearer than multiplying with the inverse.